packages/docusaurus-theme-classic/src/theme/Navbar/Layout/index.tsx

```diff
@@ -33,9 +34,15 @@ export default function NavbarLayout({children}: Props): JSX.Element {
   } = useThemeConfig();
   const mobileSidebar = useNavbarMobileSidebar();
   const {navbarRef, isNavbarVisible} = useHideableNavbar(hideOnScroll);
+  const ariaLabel = translate({
+    id: 'theme.NavBar.mainAriaLabel',
```

Comment:
```suggestion
    id: 'theme.NavBar.navAriaLabel',
```

We already have some with a similar name so let's stay consistent:

<img width="290" alt="CleanShot 2022-11-24 at 10 29 08@2x" src="https://user-images.githubusercontent.com/749374/203747585-cf3ac753-0c03-44a5-96ff-f4e53cf87108.png">

Reply:
Awesome, the `Id` has been adjusted to `theme.NavBar.navAriaLabel`.  Thanks a bunch.